trino/src/main/java/com/netease/arctic/trino/unkeyed/IcebergMetadata.java

```diff
@@ -1523,7 +1523,7 @@ public void executeRemoveOrphanFiles(ConnectorSession session, IcebergTableExecu
         IcebergSessionProperties.REMOVE_ORPHAN_FILES_MIN_RETENTION);
 
     if (table.currentSnapshot() == null) {
-      log.debug("Skipping remove_orphan_files procedure for empty table " + table);
+      log.debug("Skipping remove_orphan_files procedure for empty table ", table);
```

Comment:
According to the `io.airlift.log` logging framework used by Trino, the format code needs to be modified to `Skipping remove_orphan_files procedure for empty table %s`.

Reply:
done.

Reply:
I rollback class HiveMetaStoreClient.java, because the class was copied from hive 2.1.1. I think we should not change it.